src/clj/nnts2/db/organization.clj

```diff
@@ -18,3 +18,29 @@
                                  sql/format)
                    {:identifiers utils/snake->kebab})
        first)))
+
+(defn add-user
+  ([data] (add-user data config/db-spec))
+  ([data db-spec]
+   (let [casted-data (assoc data :role (sql/call :cast (:role data) :role))]
+     (-> (jdbc/query (db-spec)
+                     (-> (h/insert-into :members)
+                         (h/values [casted-data])
+                         (ph/on-conflict-constraint :unique_org_id_user_id)
+                         (ph/do-nothing)
+                         (ph/returning :*)
+                         sql/format)
+                     {:identifiers utils/snake->kebab})
+         first))))
+
+(defn get-by-user-id
+  ([user-id] (get-by-user-id user-id config/db-spec))
+  ([user-id db-spec]
+   (jdbc/query (db-spec)
+               (-> (h/select :name :slug :org.id)
+                   (h/from [:members :m])
+                   (h/join [:organizations :org] [:= :org.id :m.org-id])
+                   (h/where [:= :m.user-id user-id])
+                   sql/format)
+               {:identifiers utils/snake->kebab})))
+
```

Comment:
Instead of having very specific functions to get data, a generic get on membership table should exist where you can do:
- get users of  a specific organization
- get orgs for a specific user
- get admin roles for an org 
etc.